### DISCUSSION TITLE
Cannot match .dat files in database/objects with actual data libraries filenames.

### DISCUSSION THREAD

**Discussion by malouan** [galaxyproject/galaxy#18611] [Technical Help / Q&A]
Hi all.
I need to match .dat filenames with the actual filenames in the data libraries. The Galaxy server is crashed and I need to move data libraries on the new server. 

Any ideas?
Thanks!
AM

**Comment by mvdbeek** [comment]
Metadata for your files are stored in your database. You will need to be able to access the database to get any of these. Your best bet is to transfer the database to your new server or write some SQL to join against the history or library datasets. This is not straightforward though as a single dataset might have multiple related entries in the history_dataset_association or library_dataset_dataset_association.
